mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -85,6 +85,18 @@ class LinearRegression(override val uid: String)
   setDefault(fitIntercept -> true)
 
   /**
+   * Whether to standardize the training features before fitting the model.
+   * The coefficients of models will be always returned on the original scale,
+   * so it will be transparent for users. Note that when no regularization,
+   * with or without standardization, the models should be always converged to
+   * the same solution.
+   * Default is true.
+   * @group setParam
+   */
+  def setStandardization(value: Boolean): this.type = set(standardization, value)
+  setDefault(standardization -> true)
```

Comment:
If an update does not change the default behavior, I don't think there's a great need to notify the user.  (Unnecessary documentation detracts from important docs.)  But I do like the idea of saying somewhere that the default behavior matches that of glmnet (in the Scala doc of setStandardization).